src/plugins/home/public/application/components/home.test.tsx

```diff
@@ -31,46 +33,31 @@ jest.mock('../../../../../../src/plugins/kibana_react/public', () => ({
 }));
 
 describe('home', () => {
-  let defaultProps;
+  let defaultProps: HomeProps;
 
   beforeEach(() => {
     defaultProps = {
       directories: [],
       solutions: [],
-      apmUiEnabled: true,
-      mlEnabled: true,
-      kibanaVersion: '99.2.1',
-      fetchTelemetry: jest.fn(),
-      getTelemetryBannerId: jest.fn(),
-      setOptIn: jest.fn(),
-      showTelemetryOptIn: false,
-      addBasePath(url) {
-        return `base_path/${url}`;
-      },
-      find() {
-        return Promise.resolve({ total: 1 });
-      },
-      loadingCount: {
-        increment: sinon.mock(),
-        decrement: sinon.mock(),
-      },
```

Comment:
the moment you switch to TS and realize that the tests are not actually testing the logic 🤯 😅